### DISCUSSION TITLE
About multisite installation

### DISCUSSION THREAD

**Discussion by alvarpb** [globaleaks/globaleaks-whistleblowing-software#3567] [Q&A]
We have to deploy a multisite installation, but the question we have is if instead of configuring the different entities one by one, we would like to know if we can create one, and since they will all be exactly the same, we could clone them in a simple way.

Thanks,

**Comment by evilaliv3** [comment]
Thank you for you question.

There is some plan for implementing a profile based configuration enabling
to create configurations and replicate them but there is currently no
schedule for release as the feature is moved forward best effort

Here you could find details about what we discussed:
https://github.com/globaleaks/GlobaLeaks/issues/2039

It would be great if you could check the ticket and comment over it
